src/plugins/data_view_management/public/management_app/data_view_mgmt_selectors.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { DataViewMgmtState } from './data_view_management_service';
+
+export const dataViewSelector = (state: DataViewMgmtState) => state.dataView;
+export const allowedTypesSelector = (state: DataViewMgmtState) => state.allowedTypes;
```

Comment:
Might be always `[]` with the current implementation.

Reply:
this has been fixed, it is used